### PR TITLE
feat(Core/Commands): add honor and arena point reset to `.reset all` …

### PR DESCRIPTION
**Pull request by Nyeriah**
…command (#20)

* Initial plan

* feat(Core/Commands): add honor and arena to reset all command

This pull request adds support for resetting honor and arena points for all players via the `.reset all honor` and `.reset all arena` commands. It introduces new database statements, updates command help text, and adds localized messages for these actions.

**Command functionality enhancements:**

* Added support for `.reset all honor` and `.reset all arena` commands, enabling administrators to reset honor and arena points for all players. This includes updating both the database and in-memory values for online players, and broadcasting appropriate messages. (`src/server/scripts/Commands/cs_reset.cpp`)

**Database updates:**

* Introduced new prepared statements `CHAR_UPD_ALL_HONOR_POINTS` and `CHAR_UPD_ALL_ARENA_POINTS` to reset honor and arena points for all characters in the database. (`src/server/database/Database/Implementation/CharacterDatabase.cpp`, `src/server/database/Database/Implementation/CharacterDatabase.h`) [[1]](diffhunk://#diff-642c906075729a1eb0ec3dcdcf01e08de47d00ca88464b74b876f14f468f086eR505-R506) [[2]](diffhunk://#diff-3f9e61636f74100b903abddf7d5805c185e0b5f814979de53ebc1868a53b3fe3R429-R430)

**Localization and messaging:**

* Added new string entries (`LANG_RESETALL_HONOR`, `LANG_RESETALL_ARENA`) to support localized messages when honor or arena points are reset. (`src/server/game/Miscellaneous/Language.h`, `data/sql/updates/pending_db_world/rev_2026_03_03_014716.sql`) [[1]](diffhunk://#diff-54db525180fde70d66014a863c330b6bddb2978fd47873d1b11bf8fb92b6ac98L1186-R1189) [[2]](diffhunk://#diff-4727692c9e9b4c8b979333c87c22d0fae37346f49936b638ab0057be25cae38aR1-R8)
* Updated the help text for the `reset all` command to include instructions for honor and arena resets. (`data/sql/updates/pending_db_world/rev_2026_03_03_014716.sql`)

---------

## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [ ] Database (SAI, creatures, etc).

### AI-assisted Pull Requests

> [!IMPORTANT]
> While the use of AI tools when preparing pull requests is not prohibited, contributors must clearly disclose when such tools have been used and specify the model involved.
> 
> Contributors are also expected to fully understand the changes they are submitting and must be able to explain and justify those changes when requested by maintainers.

- [x] AI tools (e.g. ChatGPT, Claude, or similar) were used entirely or partially in preparing this pull request. Please specify which tools were used, if any.

## Issues Addressed:

- Closes 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: src/server/game/Miscellaneous/Language.h [hunk 1]
```diff
@@ -1182,7 +1182,7 @@ enum AcoreStrings
     LANG_CMD_QUEST_STATUS_CONDITION     = 5108,
     LANG_CMD_QUEST_STATUS_LOG_FULL      = 5109,
     LANG_CMD_QUEST_STATUS_COND_DETAIL   = 5110,
-
+
```
Comment (Nyeriah) [inline src/server/game/Miscellaneous/Language.h]:
```suggestion

```
